crates/chain/src/keychain/txout_index.rs

```diff
@@ -200,14 +176,11 @@ impl<K: Clone + Ord + Debug> KeychainTxOutIndex<K> {
     /// Set the lookahead count for `keychain`.
     ///
     /// The lookahead is the number of scripts to cache ahead of the last stored script index. This
-    /// is useful during a scan via [`scan`] or [`scan_txout`].
+    /// is useful during a scan via [`Indexer::index_tx`] or [`Indexer::index_txout`].
```

Comment:
Since we are not necessarily encouraging people to "scan" things with these methods (it's mostly done automatically) I don't think this sentence is doing much for us. Instead explain when it is useful at a higher level i.e. when you are processing block data and want to find outputs you own beyond your last revealed index because the last revealed index may not be accurate (some other system may have revealed script pubkeys beyond it).